src/main/java/org/opensearch/knn/jni/JNIService.java

```diff
@@ -99,14 +99,16 @@ public static long loadIndex(String indexPath, Map<String, Object> parameters, S
      * @param k            neighbors to be returned
      * @param engineName   name of engine to query index
      * @param filteredIds  array of ints on which should be used for search.
+     * @PARAM filterIdsType how to filter ids: Batch or BitMap
```

Comment:
this is invalid tag and failing the build. Please fix this.

Reply:
Thanks for the review, all fixed at https://github.com/opensearch-project/k-NN/pull/1402/commits/568972fc3dce704bab6dc26b356cb37ef54de1e3.